libblkid-rs-sys/build.rs

```diff
@@ -1,11 +1,36 @@
 use bindgen::Builder;
 
+use cfg_if::cfg_if;
 use std::{env, path::PathBuf};
 
+cfg_if! {
+    if #[cfg(feature = "pkg-config")] {
+        fn setup_bindings_builder() -> Builder {
+            let mut bindings_builder = Builder::default();
+            let mut pkg_config = pkg_config::Config::new();
+            let pkg_config = pkg_config.atleast_version("2.33.2");
+            #[cfg(feature = "static")]
+            {
+                pkg_config.statik(true);
+            }
+            let libblkid = pkg_config.probe("blkid").expect("Failed to find libblkid?");
+
+            for include in libblkid.include_paths {
+                bindings_builder = bindings_builder.clang_arg(format!("-I{}", include.display()));
+            }
+            bindings_builder
+        }
+    }else {
+        fn setup_bindings_builder() -> Builder {
+            Builder::default()
+        }
+    }
+}
+
 fn main() {
     println!("cargo:rustc-link-lib=blkid");
```

Comment:
Do we need to do something about this too? Like add the library search paths? Otherwise I don't see how the "static" feature would take effect.

Reply:
The pkg-config prints its own `cargo:rustc-link-lib` statement, though I have moved that print to only be printed on non-static.